conf/test_full.config

```diff
@@ -23,3 +23,22 @@ params {
     intervals   = 's3://ngi-igenomes/test-data/sarek/S07604624_Padded_Agilent_SureSelectXT_allexons_V6_UTR.bed'
     wes         = true
 }
+
+process {
+    // Request a GPU for every parabricks process
+    // NOTE https://docs.nvidia.com/clara/parabricks/latest/gettingstarted.html#hardware-requirements
+    withName: 'PARABRICKS_.*' {
+        accelerator    = 4
+        // NOTE Borrowed from https://github.com/clara-parabricks-workflows/parabricks-nextflow/blob/dce38f46e46e8595223e4bcaee2eca6fe94865e6/conf/genome.config#L15C1-L31C6
+        cpus           = { 48 * task.attempt }
+        memory         = { task.attempt > 1 ? '370.GB' : '186.GB' }
+        time           = { 1.h * task.attempt }
+        resourceLimits = [cpus: 96, memory: 370.GB]
+        maxRetries = 3
+        errorStrategy = { task.exitStatus in [1,2,143,137,104,134,139,255] ? 'retry' : 'finish' }
```

Comment:
we ar eloosing the snapshot retry here. Do we need all these error codes @edmundmiller ?